File: 3/programming/week3/30-22126-nishiyama_sou-kadai-3.c
#include <stdio.h>

int main(void){
	struct Human{
		char name[100];
		int age;
	};

	struct Human human;
	printf("氏名：");
	scanf("%s", human.name);
	printf("年齢：");
	scanf("%d", &human.age);

	//int miseinen = (human.name > 17) ? 1: 0;
	printf("%sは%sです！\n", human.name, human.age > 17 ? "成年": "未成年");
	return 0;
}

File: 3/programming/week3/30-22126-nishiyama_sou-kadai-3.py
class Human:
	name = ""
	age = 0
human = Human
human.name = input("氏名：")
human.age = int(input("年齢："))
miseinen = "成年" if human.age >= 18 else "未成年"
print(f'{human.name}は{miseinen}です！')
